mula/scheduler/schedulers/scheduler.py

```diff
@@ -192,9 +194,9 @@ def push_item_to_queue_with_timeout(self, item: models.Task, max_tries: int = 5,
         if tries >= max_tries and max_tries != -1:
             raise QueueFullError()
 
-        self.push_item_to_queue(item)
+        self.push_item_to_queue(item=item, create_schedule=create_schedule)
```

Comment:
For consistency with the rest of the code, I suggest using positional arguments instead of keyword arguments where possible. This aligns better with the overall style and helps keep the lines concise. (Of course, there are cases where passing arguments as keywords only is preferable, but in this case the method signature is relatively simple)

```suggestion
        self.push_item_to_queue(item, create_schedule=create_schedule)
```